### DISCUSSION TITLE
Using IPFS Discord as default chat channel

### DISCUSSION THREAD

**Discussion by bmann** [wnfs-wg/spec#17] [General]
I’ve arranged to create a [#wnfs channel in the IPFS Discord chat](https://discord.gg/vj7qWuAyHY) as the companion to this discussion forum. 

We should mention it in the README and elsewhere. 

cc @b5 @expede 

and @RangerMauve who prompted this ;)

**Comment by matheus23** [comment]
We decided in #30 that that'd be a better fit for the organization as a whole:
https://github.com/wnfs-wg
![image](https://user-images.githubusercontent.com/1430958/184604786-675a4a64-cb78-42f9-b4b1-b637404cc79d.png)
